cobalt/build/BUILD.gn

```diff
@@ -13,15 +13,9 @@
 # limitations under the License.
 
 action("cobalt_build_id") {
-  cobalt_version = exec_script("get_build_id.py", [], "trim string")
-
   script = "build_id.py"
   outputs = [ "$root_gen_dir/cobalt_build_id.h" ]
-
-  args = [
-    rebase_path(outputs[0], root_build_dir),
-    cobalt_version,
-  ]
+  args = [ rebase_path(outputs[0], root_build_dir) ]
```

Comment:
Note that this change changes the time of build id generation from the GN step to the Ninja step, which can have different results. afaik it won't affect anything, but something to keep in mind

Reply:
That should be fine. I just doubled checked this, but it seems that this isn't the case. When I run the gn step neither the cobalt_build_id.h nor the build_info.json files are generated until the ninja step.

Reply:
It is the case—while the cobalt_build_id.h file isn't yet generated after running GN, the cobalt_version is already set. The preset cobalt_version is written to cobalt_build_id.h at compilation (ninja) time.

Reply:
Ah! Gotcha, though I think it being set and generated at the same step now is easier to follow. Either way I don't see us running into any issues with it being set 1 step later now.